test/local/routes/emails.js

```diff
@@ -656,8 +656,8 @@ describe('/recovery_email', () => {
 
   describe('/recovery_email', () => {
```

Comment:
Fwiw, missing tests for logic inside route handlers is now an official bugbear for me post-#1980.